config/routes.rb

```diff
@@ -1,8 +1,9 @@
 Rails.application.routes.draw do
   devise_for :users
   root to: 'pages#home'
-  resources :activities do
-    resources :bookings
+
+  resources :activities, only: [:index, :new, :show, :create] do
```

Comment:
Need to implement the 'Destroy/Edit' methods, we can use current_user == user_id to authenticate the author of the 'Activity' post or use 'Pundit'.